### ISSUE TITLE
Find-as-you-type keyboard navigation doesn't work with extended characters

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#1170]
https://forums.zotero.org/discussion/comment/269491/#Comment_269491

Not sure how easy this would be to fix.
